### PR TITLE
Licence modal v3

### PR DESCRIPTION
**Pull request by flatsiedatsie**

### CODE REVIEW

File: static/js/views/discovered-addon.js [hunk 1]
```diff
@@ -123,6 +129,60 @@ class DiscoveredAddon {
...
+            () => {
+              modal = document.getElementById("media-modal");
+              modal.parentNode.removeChild(modal);
+            }
+          );
+
+          fetch(licenseUrl).then(response => {
+            if (!response.ok) {
+              throw new Error("HTTP error " + response.status);
+              document.getElementById("media-modal-text").innerText = fluent.getMessage('failed-read-file');
```
Comment (tim-hellhake) [inline static/js/views/discovered-addon.js:171]: That's dead code

File: static/js/views/installed-addon.js [hunk 2]
```diff
@@ -204,12 +209,12 @@ class InstalledAddon {
...
-        if (this.metadata.content_scripts &&
-            this.metadata.content_scripts.length > 0 &&
-            this.enabled) {
-          window.location.reload();
+        // If this add-on is a UI extension, reload the page to pick up the	
+        // changes	
+        if (this.metadata.content_scripts &&	
+            this.metadata.content_scripts.length > 0 &&	
+            this.enabled) {	
+          window.location.reload();
```
Comment (tim-hellhake) [inline static/js/views/installed-addon.js:217]: I don't see any real changes here.
You might want to revert this.

Comment (tim-hellhake) [inline static/js/views/installed-addon.js:217]: I don't see any real changes here.
You might want to revert this.

File: static/fluent/en-US/main.ftl [hunk 3]
```diff
@@ -629,3 +630,4 @@ edit-button =
     .aria-label = Edit
 save-button =
     .aria-label = Save
+    .aria-label = Save
```
Comment (tim-hellhake) [inline static/fluent/en-US/main.ftl:633]: Looks like a duplicate

File: static/js/views/discovered-addon.js (hunk 1 above)
Comment (flatsiedatsie) [reply static/js/views/discovered-addon.js:171]: Could you be more specific?

File: static/fluent/en-US/main.ftl (hunk 3 above)
Comment (flatsiedatsie) [reply static/fluent/en-US/main.ftl:632]: How did that happen.. fixed

File: static/js/views/discovered-addon.js (hunk 1 above)
Comment (flatsiedatsie) [reply static/js/views/discovered-addon.js]: I've removed the check for an ok response.

**Pull request closed without merging**